subsys/spm/spm.c

```diff
@@ -39,9 +39,9 @@
 #define NON_SECURE_RAM_OFFSET 0x10000
 #endif /* USE_PARTITION_MANAGER */
 
-#define NON_SECURE_FLASH_REGION_INDEX \
+#define NON_SECURE_FLASH_REGION_INDEX                                          \
```

Comment:
Are these intended?

Reply:
No. I used the an auto-format function and it must have done some unintended changes. Will fix it.